enterprise/models.py

```diff
@@ -1107,6 +1110,74 @@ class Meta:
         app_label = 'enterprise'
         ordering = ['created']
 
+    @classmethod
+    def link_pending_enterprise_user(cls, pending_ecu, is_user_created, user):
+        """
+        Link a PendingEnterpriseCustomerUser to the appropriate EnterpriseCustomer by
+        creating a EnterpriseCustomerUser record.
+
+        Arguments:
+            pending_ecu: a PendingEnterpriseCustomerUser instance
```

Comment:
why pass in pending_ecu as an argument when this method is in its class? why not just call this method on the pending_ecu?